spirv_glsl.hpp

```diff
@@ -64,6 +65,149 @@ typedef uint32_t AccessChainFlags;
 
 class CompilerGLSL : public Compiler
 {
+	struct KHR_shader_subgroupSupportHelper
```

Comment:
Move this below `private:`. Also, in the public header, avoid the inlined definitions to keep the code more compact.

Reply:
The class names should be pascal case, i.e. ShaderSubgroupSupportHelper. Don't need the KHR.